implementing-nosto/implement-search/implement-search-using-code-editor/implementing-search-page.md

```diff
@@ -204,14 +199,68 @@ The supported sizes are
   </tbody>
 </table>
 
+The same mapping will also be attempted for SKU level data
 
-The `priceDecorator` uses the currency formatting definitions of the Nosto account to format prices into `priceText` and `listPriceText` fields. The fields required for this mapping are
+### Price decorators
 
-* `price` will be formatted to `priceText`
-* `listPrice` will be formatted to `listPriceText`
-* `priceCurrencyCode` will be used as the currency code
+* **Include Required Fields**
+  The `priceDecorator` uses the currency formatting definitions of the Nosto account to format prices into `priceText` and `listPriceText` fields. The fields required for this mapping are:
+  * `price` will be formatted to `priceText`
+  * `listPrice` will be formatted to `listPriceText`
+  * `priceCurrencyCode` will be used as the currency code
+* **Use the `priceDecorator`**
+  The `priceDecorator` is responsible for formatting prices into text fields using above mentioned fields.
+ 
+A complete example of the Search-templates configuration:
 
-The same mapping will also be attempted for SKU level data
+```javascript
+import { init, priceDecorator } from "@nosto/preact";
+
+init({
+    ...window.nostoTemplatesConfig,
+    ...
+    serpQuery {
+        products: {
+            variationId: this.variationId(),
+            fields: [
+                // needed for priceDecorator
+                "price", 
+                "listPrice",
+                "priceCurrencyCode",
+            ],
+            size: 20,
+            from: 0
+        }
+    },
+    hitDecorators: [
+        priceDecorator()
+    ]
+});
+```
+> For the `priceDecorator` to function correctly, the Multi-Currency setup outlined below must be properly configured.
+
+### Multi-Currency
+
+To enable multi-currency functionality in search templates, follow these steps:
+
+* **Enable Multi-Currency in Nosto Admin** - [Enabling multi-currency from the admin](https://docs.nosto.com/techdocs/apis/frontend/implementation-guide-session-api/advanced-usage/spa-adding-support-for-multi-currency#enabling-multi-currency-from-the-admin)
+
+* **Provide the `variationId`**  
+  The `variationId` is essential for converting prices to the correct currency. It should be included in the search query to ensure accurate price conversion. Below is an example of how to include the `variationId` in your search query:
```

Comment:
I would maybe express that the variationId is used to express the response price data currency